server/openapi/model_test_1.go

```diff
@@ -0,0 +1,43 @@
+/*
+ * TraceTest
+ *
+ * OpenAPI definition for TraceTest endpoint and resources
+ *
+ * API version: 0.2.1
+ * Generated by: OpenAPI Generator (https://openapi-generator.tech)
+ */
+
+package openapi
+
+type Test1 struct {
```

Comment:
This is not looking great. I guess this is because it already has a `Test` struct defined, right?
Maybe we can name this `FileTest` or use some kind of prefix/suffix so our naming is more self explainatory